test_databaseconnect.py

```diff
@@ -1,86 +1,28 @@
 import databaseconnect
-import config
-import pytest
-import mysql.connector as mysql
 
 
 class TestClass:
-    @pytest.fixture(scope="session", autouse=True)
-    def setup_init(self):
-        # Will be executed before the first test
-        main_database = config.database
-        config.database = "test"
-        try:
-            test_db = mysql.connect(
-                host=config.host,
-                user=config.user,
-                passwd=config.password,
-                database=config.database,
-            )
-
-            cursor = test_db.cursor()
-            cursor.execute("CREATE DATABASE {}".format(config.database))
-            print("test database created")
-
-        except Exception:
-            print("Failed to create test database")
-            # rolling back to main db
-            config.database = main_database
-            pytest.exit("Exiting test!")
-
-        yield test_db
-
-        # Will be executed after the last test is executed
-        try:
-            mycursor = test_db.cursor()
-            mycursor.execute("DROP DATABASE {}".format(config.database))
-            mycursor.close()
-            print("test database deleted.")
-
-        except Exception:
-            print("Failed to delete test database.")
-
-        config.database = main_database
-
     def test_setup_database(self):
-        db = databaseconnect.setup_database()
-        cursor = db.cursor()
-        cursor.execute("SHOW TABLES")
-        tables = cursor.fetchall()
+        tables = databaseconnect.setup_database()
         expected_tables = [
-            ("chat_table"),
-            ("statement_table"),
-            ("question_table"),
-            ("directions_table"),
+            ("chat_table",),
+            ("directions_table",),
+            ("question_table",),
+            ("statement_table",),
         ]
-        assert tables.sort() == expected_tables.sort()
+        assert sorted(tables) == sorted(expected_tables)
 
     def test_add_to_database_of_chat_table(self):
-        db = databaseconnect.add_to_database("C", "subject", "root", "verb", "H")
-        cursor = db.cursor()
-        cursor.execute(
-            "select * from chat_table where root_word='root' and verb='verb' and sentence='H'"
-        )
-        res = cursor.fetchone()[0]
-        assert res == 1
+        res = databaseconnect.add_to_database("C", "subject", "root", "verb", "H")
+        assert res == "Success"
 
     def test_add_to_database_of_question_table(self):
-        db = databaseconnect.add_to_database("Q", "subject", "root", "verb", "H")
-        cursor = db.cursor()
-        cursor.execute(
-            "select * from question_table where subject='subject' and root_word='root' and verb='verb' and sentence='H'"
-        )
-        res = cursor.fetchone()[0]
-        assert res == 1
+        res = databaseconnect.add_to_database("Q", "subject", "root", "verb", "H")
+        assert res == "Success"
```

Comment:
Need your help to create better test statements that will check against SQL Injection and the issue faced in #62. @janakrajchadha